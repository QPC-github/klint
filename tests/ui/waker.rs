#![crate_type="lib"]

#[klint::preempt_count(expect = 0..)]
fn waker_ops(x: &core::task::Waker) {
    x.clone().wake();
    x.wake_by_ref();
}
