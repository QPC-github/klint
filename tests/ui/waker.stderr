WARN klint::ctxt no klint metadata found for crate core
